io.github.mrstickypiston.stickyhours.yml

```diff
@@ -0,0 +1,90 @@
+app-id: io.github.mrstickypiston.stickyhours
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: stickyhours
+cleanup:
+  - /include
+  - /share/man
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=fallback-x11
+modules:
+  - name: cpython
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/bin
+      - cp --archive python/bin/* /app/bin
+      - chmod -R +x /app/bin
+      - mkdir -p /app/include
+      - cp --archive python/include/* /app/include
+      - mkdir -p /app/lib
+      - cp --archive python/lib/* /app/lib
+      # Update installation prefix for pip package builds
+      - sed --follow-symlinks -i 's/prefix="\/install"/prefix="\/app"/g' /app/bin/python3-config
+      - sed --follow-symlinks -i 's/prefix=\/install/prefix=\/app/g' /app/lib/pkgconfig/python3.pc /app/lib/pkgconfig/python3-embed.pc
+    sources:
+      - type: archive
+        url: https://github.com/MrStickyPiston/StickyHours/releases/download/1.0.3/StickyHours-Flatpak-src.zip
+        sha256: 864af4bb1607ce8f37bc26243c2845f72f40656f833caa8a326e2d0d97845eee
+        dest: .
+    cleanup:
+      - /bin/2to3*
+      - /bin/idle*
+      - /lib/python*/config-*
+
+  - name: bootstrap
+    no-autogen: true
+    sources:
+      - type: archive
+        url: https://github.com/MrStickyPiston/StickyHours/releases/download/1.0.3/StickyHours-Flatpak-src.zip
```

Comment:
this use the same source as the previous module. why is it separate?

Reply:
i thought it would become a mess if it was not seperated, and flatpak doesnt download the same file multiple times so it does not really hurt performance

Reply:
it does and it's confusing.

Reply:
also this one is different because it builds the c project and that needs to be in the root dir

Reply:
fixed it by using my own bootstrap